File: src/app-client/chan-at.js
function chan_at(x,y)
{
 check(is_arr,x)
 check(is_uint,y)
 check(lt,y,dimension)
 
 const r=arr()
 
 for(const k in x)
 {
  const i=to_i(k)
  const v=at(x,i)
  const n=at(v,y)
  
  push(r,n)
 }
 
 return r
} 

File: src/spa-jneeg-client/chan-at.cs
fn chan_at x y
 check is_arr x
 check is_uint y
 check lt y dimension
 
 let r arr
 
 forin x
  let i to_i k
  let v at x i
  let n at v y
  
  push r n
 end
 
 ret r
end
